LICENSE

```diff
@@ -1,3 +1,7 @@
+All changes to LLVM-CBE contributed by Julia Computing are made available under
+the original University of Illinois/NCSA Open Source License.
+Copyright(c) 2015-2021 Julia Computing.
+
```

Comment:
Nitpick: that this is the old LLVM licence is obvious from the rest of the file, so maybe this statement is more useful if placed in the README instead?

(I have nothing useful to say about the copyright line.)